integrations-and-supported-tools/keras/scripts/Neptune_Keras_more_options.py

```diff
@@ -22,15 +22,15 @@
 
 params = {"lr": 0.005, "momentum": 0.9, "epochs": 10, "batch_size": 32}
 
-# log hyper-parameters
+# (Neptune) log hyper-parameters
 run["hyper-parameters"] = params
 
 optimizer = tf.keras.optimizers.SGD(learning_rate=params["lr"], momentum=params["momentum"])
 
 model.compile(optimizer=optimizer, loss="sparse_categorical_crossentropy", metrics=["accuracy"])
 
-# log metrics during training
-neptune_cbk = NeptuneCallback(run=run, base_namespace="metrics")
+# (Neptune) log metrics during training
+neptune_cbk = NeptuneCallback(run=run)
```

Comment:
Latest version doesn't require passing `base_namespace` and defaults to training.